playbooks/adhoc/uninstall.yml

```diff
@@ -39,7 +39,7 @@
         - atomic-openshift-master-api
         - atomic-openshift-master-controllers
         - atomic-openshift-node
-        - etcd
+        - "{% if is_atomic %}etcd_container{% else %}etcd{% endif %}"
```

Comment:
In this case, we list out all services that could be encountered...  instead of using the conditional here, it would probably be better to just list etcd_container separately.

As an aside, instead of using `{% if is_atomic %}...{% endif %}` here, you could have done `{{ etcd_container if is_atomic | bool else etcd }}` (The use of the bool filter just ensures that is_atomic is cast as a boolean before comparing in case it is a string representation of a boolean)